Chapter 6 - physical layer (#39)
UAVCAN/specification: specification/physical_layer/can.tex

Other connector types may be added in future revisions of the specification.

It is highly recommended to provide two identical parallel connectors for each CAN interface per device,

> so that the device can be connected to the bus without the need to use \mbox{T-connectors}.

Reviewer: ```suggestion
and not using \mbox{T-connectors}.
```

> \mbox{T-connectors} should be avoided when possible because generally they add an extra point of failure,

Reviewer: ```suggestion
\mbox{T-connectors} should be avoided because they add another point of failure,
```
